File: courses/codes/hw2-java-sol/HW2Q2.java
import java.util.Scanner;

public class HW2Q2 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter the first number: ");
		int x = sc.nextInt();
		System.out.print("Enter the second number: ");
		int y = sc.nextInt();
		
		double doubleX = (double) x;
		double actual = doubleX / y;
		
		int intDiv = x / y;
		int rem = x % y;
		
		System.out.println(x + " divided by " + y + " is " + actual);
		System.out.println(y + " goes into " + x + " ... " + intDiv + " times, with a remainder of " + rem);
	}

}

File: courses/codes/hw2-py-sol/hw2_q2.py
x = int(input("Enter the first number: "))
y = int(input("Enter the second number: "))

z = x // y      # integer division
w = x / y       # regular division
r = x % y       # remainder

print("%d divided by %d is %.4f" % (x, y, w))
print("%d goes into %d ... %d times, with a remainder of %d" % (y, x, z, r))